packages/nextjs/app/token-vendor/page.tsx

```diff
@@ -26,14 +25,24 @@ const TokenVendor: NextPage = () => {
 
   const { address: connectedAddress } = useAccount();
 
-  // ToDo: Add YourToken symbol
```

Comment:
We need to show the Symbol of the token same as speedrunethereum

Reply:
fixed